boards/shields/adafruit_2_8_tft_touch_v2/dts/adafruit_2_8_tft_touch_v2.dtsi

```diff
@@ -18,31 +18,38 @@
 		invert-x;
 		invert-y;
 	};
+
+	mipi_dbi {
```

Comment:
If that's possible to have 2 of those on a single build (ie board screen + shield), we need a `adafruit_2_8_tft_touch_v2_` prefix.

Reply:
Yeah, it could be (one MIPI DBI emulated driver would be needed to wrap each SPI controller instance with a display connected), so I'll add the prefix